composefiles/swarm-uniflow-krakend.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
```

Comment:
why is it moved to new file? I prefer to keep it where it was if possible

Reply:
the image pull fails because the tag doesn't exists